jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -856,7 +856,7 @@ void java_bytecode_convert_classt::add_array_types(symbol_tablet &symbol_table)
     code_blockt clone_body;
 
     code_declt declare_cloned(local_symexpr);
-    clone_body.move_to_operands(declare_cloned);
+    clone_body.move(declare_cloned);
```

Comment:
Move the declaration of `clone_body` further down and do `code_blockt clone_body({declare_cloned, create_blank, declare_index, copy_loop, return_inst});` (if I got this right from what I see in the code review).